slides/bitcoin-primer.tex

```diff
@@ -0,0 +1,216 @@
+% Choose one to switch betweeen slides and handout
+\documentclass[]{beamer}
+%\documentclass[handout]{beamer}
+
+% Video Meta Data
+\title{Bitcoin, Blockchain and Cryptoassets}
+\subtitle{Bitcoin Primer}
+\author{Prof. Dr. Fabian Schär}
+\institute{University of Basel}
+
+% Config File
+\input{../config/config.tex}
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+\begin{document}
+
+\thispagestyle{empty}
+\begin{frame}[noframenumbering]
+	\titlepage
+\end{frame}
+
+%%%
+\begin{frame}{Definition}
+Some key aspects of Bitcoin: \vspace{1em}
+
+	\begin{enumerate}
+		\item<2-> Competitive creation
+		\item<3-> Virtual representation
+		\item<4-> Decentralized management
+	\end{enumerate}
+	\vspace{1em}	
+\uncover<5->{Bitcoin is a decentralized data structure.} \\	\vspace{1em}
+\uncover<6->{The system is maintained by its participants and works in the absence of centralized third parties.} 
+\end{frame}
```

Comment:
There will be a video which covers the monetary control structures (of Bitcoin) right before the Bitcoin Primer. I suggest that we then either:
- remove the Definition slide and summarize its content during the introduction of the video, or
- that we replace the bullet points on this slide with the figure from the monetary control structures slide deck.

Anyway, let's leave it as is and fix it as soon as the monetary control structure slides are ready.